rts/Lua/LuaHandle.cpp

```diff
@@ -1614,6 +1615,8 @@ void CLuaHandle::Update()
 
 	// call the routine
 	RunCallIn(L, cmdStr, 0, 0);
+
+	LuaVFSDownload::ProcessDownloads();
```

Comment:
Is this supposed to happen after every Update of every LuaHandle or just once every Update event?